config.md

```diff
@@ -390,20 +405,166 @@ Here is a full example `config.json` for reference.
     "hooks": {
         "prestart": [
             {
-                "path": "/usr/bin/uptime",
+                "path": "/usr/bin/fix-mounts",
                 "args": [
-                    "/usr/bin/uptime"
+                    "fix-mounts",
+                    "arg1",
+                    "arg2"
                 ],
-                "env": []
+                "env": [
+                    "key1=value1"
+                ]
+            },
+            {
+                "path": "/usr/bin/setup-network"
+            }
+        ],
+        "poststart": [
+            {
+                "path": "/usr/bin/notify-start",
+                "timeout": 5
+            }
+        ],
+        "poststop": [
+            {
+                "path": "/usr/sbin/cleanup.sh",
+                "args": [
+                    "cleanup.sh",
+                    "-f"
+                ]
             }
         ]
     },
     "linux": {
+        "devices": [
+            {
+                "path": "/dev/fuse",
+                "type": "c",
+                "major": 10,
+                "minor": 229,
+                "fileMode": 438,
+                "uid": 0,
+                "gid": 0
+            },
+            {
+                "path": "/dev/sda",
+                "type": "b",
+                "major": 8,
+                "minor": 0,
+                "fileMode": 432,
+                "uid": 0,
+                "gid": 0
+            }
+        ],
+        "sysctl": {
+            "net.ipv4.ip_forward": "1",
+            "net.core.somaxconn": "256"
+        },
+        "cgroupsPath": "/myRuntime/myContainer",
         "resources": {
+            "network": {
+                "classID": 1048577,
+                "priorities": [
+                    {
+                        "name": "eth0",
+                        "priority": 500
+                    },
+                    {
+                        "name": "eth1",
+                        "priority": 1000
+                    }
+                ]
+            },
+            "pids": {
+                "limit": 32771
+            },
+            "hugepageLimits": [
+                {
+                    "pageSize": "2MB",
+                    "limit": 9223372036854772000
+                }
+            ],
+            "oomScoreAdj": 100,
+            "memory": {
+                "limit": 536870912,
+                "reservation": 0,
+                "swap": 0,
```

Comment:
done

On Fri, Apr 8, 2016 at 1:06 PM, Mrunal Patel <notifications@github.com>
wrote:

> In config.md
> <https://github.com/opencontainers/runtime-spec/pull/370#discussion_r59057195>
> :
>
> > +                ]
> > +            },
> > +            "pids": {
> > +                "limit": 32771
> > +            },
> > +            "hugepageLimits": [
> > +                {
> > +                    "pageSize": "2MB",
> > +                    "limit": 9223372036854772000
> > +                }
> > +            ],
> > +            "oomScoreAdj": 100,
> > +            "memory": {
> > +                "limit": 536870912,
> > +                "reservation": 0,
> > +                "swap": 0,
>
> Perhaps make the swap same as the limit?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/370/files/e8ba1d05e1347edc6771a8133a4c729db6df3a50#r59057195>
>